Dockerfile

```diff
@@ -0,0 +1,25 @@
+# ----- Base Java - Check Dependencies ----
+FROM maven:3.6.1-jdk-8 AS base
+ENV HOME=/app
+WORKDIR $HOME
+ADD pom.xml $HOME
+
+#
+# ----Build App with Dependencies ----
+FROM base AS dependencies
+RUN mvn verify clean --fail-never
+ADD . $HOME
+RUN mvn clean package -DskipTests
+
+#
+# ---- Release App ----
+FROM  openjdk:8-jre-alpine AS release
+ENV HOME=/app
+WORKDIR $HOME
+# Create a user
+RUN addgroup -g 4120 -S helium && \
+    adduser -u 4120 -S helium -G helium
+USER helium
```

Comment:
The current version of helium has regressed back to 0.1 so the below helium jar should be helium-0.1.jar for the build to succeed.

Reply:
Resolved by adding `build_ver` as a docker build argument. As you bump versions we will not need to change the Dockerfile. Just pass the new value in the Docker Build. I will need to change the CI-CD to account for this, but is best practice.
`docker build --build-args build_ver=0.1 -t retaildevcrew/helium-java:beta .` works as expected